src/Tribe/Google_Data_Markup.php

```diff
@@ -56,27 +52,16 @@ private function build_data() {
 	 * @return string
 	 */
 	public function script_block() {
-		$events_data = $this->build_data();
-		$html        = '';
-		if ( ! empty( $events_data ) ) {
+		$data = $this->build_data();
+		$data = $this->filter_data( $data );
+
+		$html = '';
+		if ( ! empty( $data ) ) {
 			$html .= '<script type="application/ld+json">';
-			$html .= str_replace( '\/', '/', json_encode( $events_data ) );
+			$html .= str_replace( '\/', '/', json_encode( $data ) );
```

Comment:
@bordoni It is passed through a filter 2x - once with $data and once with the script block string:

Data: 
https://github.com/moderntribe/the-events-calendar/blob/feature/39616-venue-org-json-ld/src/Tribe/Google_Data_Markup.php#L41

Script block string:
https://github.com/moderntribe/the-events-calendar/blob/feature/39616-venue-org-json-ld/src/Tribe/Template/Single_Event.php#L32